modules/git/git.go

```diff
@@ -235,75 +263,75 @@ func CheckGitVersionAtLeast(atLeast string) error {
 	return nil
 }
 
-func checkAndSetConfig(key, defaultValue string, forceToDefault bool) error {
-	stdout, stderr, err := process.GetManager().Exec("git.Init(get setting)", GitExecutable, "config", "--get", key)
-	if err != nil {
-		perr, ok := err.(*process.Error)
-		if !ok {
-			return fmt.Errorf("Failed to get git %s(%v) errType %T: %s", key, err, err, stderr)
-		}
-		eerr, ok := perr.Err.(*exec.ExitError)
-		if !ok || eerr.ExitCode() != 1 {
-			return fmt.Errorf("Failed to get git %s(%v) errType %T: %s", key, err, err, stderr)
-		}
+func configSet(key, value string) error {
+	stdout, _, err := NewCommand(DefaultContext, "config", "--get", key).RunStdString(nil)
+	if err != nil && !err.IsExitCode(1) {
+		return fmt.Errorf("failed to get git config %s, err:%w", key, err)
```

Comment:
I think we need the stderr output here too.

Space after the colon:

```suggestion
		return fmt.Errorf("failed to get git config %s, err: %w", key, err)
```

Reply:
> I think we need the stderr output here too.

No worry, stderr is included in `RunStdString().Error()` already.